rpcs3/Crypto/unself.cpp

```diff
@@ -1316,7 +1326,7 @@ bool CheckDebugSelf(const std::string& self, const std::string& elf)
 
 		// Copy the data.
 		char buf[2048];
-		while (ssize_t size = s.read(buf, 2048))
+		while (size_t size = s.read(buf, 2048)) // Did size need to be of type ssize_t?
```

Comment:
The function you pointed to has a specified return type of u64 (which I take to mean as an unsigned long), but then has a case where it may return -1. I don't know what's going on there, so I'll wait on them to comment.